fearless/Common/Services/ChainRegistry/ChainRegistry.swift

```diff
@@ -136,6 +142,31 @@ final class ChainRegistry {
 }
 
 extension ChainRegistry: ChainRegistryProtocol {
+    // TODO: - Removed after demo
```

Comment:
not sure that we can merge this to main develop branch. maybe it's make sense to create demo branch